README.md

```diff
@@ -5,11 +5,18 @@
 
 Zombie detector
 ============================
-Zombie detector is CronJob to detect resources that elapsed for a long time since deletion.
+Zombie detector is a tool to detect Kubernetes resources that remain for a long time after deletion request.
+
+## Background
+In Kubernetes, there are cases where you request the deletion of resources and it remains undeleted. (For example, failing to remove the finalizer or a bug in the Operator).
+Such resources that remain undeleted (we call them zombie resources) can cause problems, so we want to detect them.
+
+When the zombie-detector detects zombie resources, it sends information to a monitoring system such as Prometheus via Pushgateway.
+This will allow us to check the status of zombie resources on the dashboard and can make alerts.
```

Comment:
The explanation looks a bit vague, and it would be better to describe clearer.
- In Kubernetes...
  - Remove _there_ and _you_ to reduce the subjects.
  - Clarify the situation by mentioning `deletionTimestamp`.
  - "Finalizer" and "operator" is not exactly specified to one, so we need to use plural forms or indefinite articles "a".
  - Operators need clarification.
- Such resources...
  - _Such that_ here looks a bit complex syntax.
  - Problems need clarification. I just added _various_ because it looks difficult to describe what exact kind of problems occur.
- When the zombie-detector...
  - Information needs to be clarified.
  - Pushgateway should have a clear reason to use.
    - It is because the zombie-detector is a short-lived program and we need to store metrics somewhere to be scraped.
 - This will...
   - Just using "this allows us" looks fine, because this system immediately allows us to see the status and fire alerts. I felt the "will" here indicates some undescribed steps.
   - The dashboard is not specified to one, we need to use "a".

```suggestion
In Kubernetes, resources sometimes remain undeleted after it got `deletionTimestamp`.
It may occur from a failure to remove finalizers or a bug in Kubernetes operators.
These remaining resources (we call them zombie resources) can cause various problems, so we want to detect them.

The zombie-detector is a short-lived program, which inspects a cluster and lists zombie resources.
It sends metrics to a Pushgateway, and they can be scraped later.
This allows us to check the status of zombie resources on a dashboard and make alerts.
```

Reply:
Thank you for your suggestion.
I fixed it.